docs/getting-started/gs-faqs.md

```diff
@@ -33,6 +34,12 @@ No. UID2 has its own framework, which is separate from EUID. As such, paperwork
 
 Yes. Through the [Transparency and Control Portal](https://www.transparentadvertising.com/), users can opt out from being served targeted ads tied to their UID2 identity. Each request is distributed through the UID2 Opt-Out Service, and UID2 Operators make the opt-out information available to all relevant participants.
 
+#### When I send DII to UID2, does UID2 store the information?
+
+No, UID2 does not store any DII. In addition, in almost all cases, UID2 doesn't store any values at all once the [POST&nbsp;/token/generate](../endpoints/post-token-generate.md), [POST&nbsp;/token/refresh](../endpoints/post-token-refresh.md), or [POST /identity/map](../endpoints/post-identity-map.md) call is complete.
+
+A necessary exception is the case where a user has opted out. In this scenario, UID2 stores a hashed, opaque value to indicate the opted-out user. The stored value cannot be reverse engineered to recognize the DII, but can be used to identify future requests for a UID2 generated from the same DII, which are therefore denied.
```

Comment:
_cannot be reverse engineered to recognize the DII_ 

Is it clearer to say

_cannot be reverse engineered back to the original value of the DII_ 

(i don't understand the term _recognize_ here too)
?

Reply:
@sunnywu that sounds great -- done -- thx.